core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20211206_v1_migration_google_project.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="dummy" author="ci-mob" id="create_V1_MIGRATION_GOOGLE_PROJECT">
+        <addColumn tableName="V1_WORKSPACE_MIGRATION_HISTORY">
+            <column name="NEW_GOOGLE_PROJECT_ID" type="VARCHAR(254)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="NEW_GOOGLE_PROJECT_NUMBER" type="VARCHAR(254)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="NEW_GOOGLE_PROJECT_CONFIGURED" type="DATETIME">
```

Comment:
This is tricky because SQL `DATETIME` does not translate to Java's `DateTime` (instead it's a `Timestamp`) and therefore adding a type related suffix doesn't really translate. I don't particually like `DATE` either because it's not a date.